templates/vue/src/components/Toolbar/index.vue

```diff
@@ -1,17 +1,23 @@
 <template>
   <div class="toolbar">
     <tapestry-filter v-if="!showMap" style="z-index: 10;" />
-    <div v-show="canEdit || (!showMap && hasDepth)" class="slider-wrapper">
-      <review-notifications v-if="canEdit && settings.submitNodesEnabled" />
-      <settings-modal-button
-        v-if="canEdit"
-        :max-depth="maxDepth"
-      ></settings-modal-button>
-      <tapestry-depth-slider
-        v-show="!showMap && hasDepth"
-        @change="updateViewBox"
-        @change:max-depth="maxDepth = $event"
-      ></tapestry-depth-slider>
+    <div v-show="isLoggedIn" class="slider-wrapper">
```

Comment:
This is not sufficient because it will show this even if all elements inside are not showing. The condition here should ensure that this toolbar is only displayed if at least one of the elements inside will display.